test/fizzBuzz_test.js

```diff
@@ -0,0 +1,19 @@
+import { expect } from 'chai'
+import fizzBuzz from '../src/fizzBuzz'
+
+describe('fizzBuzz()', function(){
+
+  it('should be a function', function(){
+    expect(fizzBuzz).to.be.a('function')
+  })
+
+  it('take an array of numbers and replace multiples of 3 with \'Fizz\' and multiples of 5 are replaced with \'Buzz\'', function(){
```

Comment:
and replaces mutliples of 3 and 5 with 'FizzBuzz.' This is a really long test. Maybe break this up into three separate tests.